src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -238,38 +237,19 @@ void PickFirst::UpdateChildRefsLocked() {
   child_subchannels_ = std::move(cs);
 }
 
-void PickFirst::UpdateLocked(const grpc_channel_args& args,
-                             RefCountedPtr<Config> lb_config) {
+void PickFirst::UpdateLocked(UpdateArgs args) {
   AutoChildRefsUpdater guard(this);
-  const ServerAddressList* addresses = FindServerAddressListChannelArg(&args);
-  if (addresses == nullptr) {
-    if (subchannel_list_ == nullptr) {
-      // If we don't have a current subchannel list, go into TRANSIENT FAILURE.
-      grpc_error* error =
-          GRPC_ERROR_CREATE_FROM_STATIC_STRING("Missing update in args");
```

Comment:
Nope.  This check is for the case where the address argument is completely missing from channel args.  Now that we're passing it as an explicit member of the `UpdateArgs` struct, it can't be missing.  (It can still be empty, but we're handling that case elsewhere.)